Blockzilla/SettingsViewController.swift

```diff
@@ -593,7 +593,7 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
     }
 
     @objc private func whatsNewClicked() {
-        highlightsButton.tintColor = UIColor.white
+        highlightsButton.tintColor = view.currentTheme == .light ? .systemGray2 : .white
```

Comment:
Suggestion: We can use a custom init for UIColor that creates separate versions for light/dark instead of checking it at the point of use.